app/src/main/java/com/firebase/uidemo/auth/AuthUiActivity.java

```diff
@@ -150,6 +161,42 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
     @MainThread
     private void handleSignInResponse(int resultCode, Intent data) {
         if (resultCode == RESULT_OK) {
+            final String prevUid = data.getStringExtra(ExtraConstants.EXTRA_MERGE_FAILED);
+            if (prevUid != null) {
+                FirebaseDatabase.getInstance()
```

Comment:
Can we remove all this?  There shouldn't be anything referencing the DB or the chat sample in the AuthUI sample.